service/controller/resource/clusterconfigmap/desired.go

```diff
@@ -72,6 +89,7 @@ func (r *Resource) GetDesiredState(ctx context.Context, obj interface{}) ([]*cor
 			},
 			"kubernetes": map[string]interface{}{
 				"API": map[string]interface{}{
+					"CACert":         clusterCA,
```

Comment:
```suggestion
					"clusterCA":         clusterCA,
```
I see the name expected is `clusterCA`

Reply:
Good call, matches https://github.com/giantswarm/cluster-operator/blob/master/service/controller/resource/clusterconfigmap/desired.go#L80 and https://github.com/giantswarm/dex-app/blob/master/helm/dex-app/templates/dex-k8s-authenticator/configmap-customer.yaml#L51